arch/sim/Kconfig

```diff
@@ -531,16 +531,31 @@ config SIM_QSPIFLASH_PAGESIZE
 		"wrap" causing the initial data sent to be overwritten.
 		This is consistent with standard SPI FLASH operation.
 
+choice
+	prompt "Simulated Prototype of HCI"
+	default SIM_HCISOCKET
+	depends on (HOST_LINUX && SIM_WALLTIME)
+
 config SIM_HCISOCKET
 	bool "Attach Host Bluetooth"
-	default false
-	depends on (WIRELESS_BLUETOOTH && HOST_LINUX && SIM_WALLTIME)
+	depends on WIRELESS_BLUETOOTH
+	---help---
+		Attached the local bluetooth device to the simulation
+		target via HCI_CHANNEL_USER. This gives NuttX full
+		control of the device, but is abstracted from the
+		physical interface which is still handled by Linux.
```

Comment:
this just repeats the help message of the other related option. please update it to reflect what it actually does different

Reply:
Done